drivers/backup/portworx/portworx.go

```diff
@@ -1113,6 +1116,32 @@ func (p *portworx) GetBackupUID(ctx context.Context, backupName string, orgID st
 	return "", fmt.Errorf("backup with name '%s' not found for org '%s'", backupName, orgID)
 }
 
+func (p *portworx) GetBackupName(ctx context.Context, backupUid string, orgID string) (string, error) {
```

Comment:
Add a comment for this function explains what it is about

Reply:
Should I duplicate the doc string here as well, given that they are already written in backup.go?

Reply:
The doc string is present in backup.go.